src/web/client/dal/remoteFetchProvider.ts

```diff
@@ -443,6 +443,14 @@ async function createFile(
         fileContent = GetFileContent(result, attributePath, entityName, entityId);
     }
 
+    const filePath : string[] = [];
+
+    Array.from(WebExtensionContext.fileDataMap.getFileMap.entries()).forEach(([fileUri, data]) => {
```

Comment:
This does not seem right - what are you trying to do here?

Reply:
from file data map storing keys (file uri), in entity data map against its entity id

Reply:
lets use the approch we discussed  - we dont need to do this iteration on a map

Reply:
updated